q2_phylogeny/tests/test_raxml.py

```diff
@@ -25,16 +26,22 @@ class RaxmlTests(TestPluginBase):
 
     package = 'q2_phylogeny.tests'
 
+    def setUp(self):
```

Comment:
The `setUp` hook runs before each individual test, but it looks like the code presented here is probably only needed to run once before the whole test suite starts - I think you can use the [`setupClass` hook](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass), instead (which should help speed things up a bit, too).

Reply:
Oh you know, one issue with making this a classmethod, is that you won't have access to `self.temp_dir` yet. Bummer.